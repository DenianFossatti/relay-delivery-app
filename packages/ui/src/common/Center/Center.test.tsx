import { mockTestID, render, wrapper } from '../../tests';

import { Center, CenterProps } from '.';

describe(`Center`, () => {
  const props: CenterProps = {
    testID: 'Center',
  };

  const el_container = mockTestID('View', props.testID!);

  it(`render component #${el_container}`, async () => {
    const { findByTestId, toJSON } = render(<Center {...props} />, { wrapper });
    const sut = await findByTestId(el_container);
    expect(sut).toBeTruthy();
    expect(toJSON()).toMatchSnapshot();
  });
});
