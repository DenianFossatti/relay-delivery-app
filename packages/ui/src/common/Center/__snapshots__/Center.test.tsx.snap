// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Atoms/Center render component #View_Center 1`] = `
<RNCSafeAreaProvider
  onInsetsChange={[Function]}
  style={
    [
      {
        "flex": 1,
      },
      undefined,
    ]
  }
>
  <View
    accessible={false}
    dataSet={{}}
    style={
      {
        "alignItems": "center",
        "display": "flex",
        "justifyContent": "center",
      }
    }
    testID="View_Center"
  />
</RNCSafeAreaProvider>
`;
